pkgs/by-name/ae/aerospace/package.nix

```
@@ -9,7 +9,7 @@
 
 let
   appName = "AeroSpace.app";
-  version = "0.17.1-Beta";
+  version = "0.18.1-Beta";
```

update to the latest version
```suggestion
  version = "0.18.2-Beta";
```